client/src/common/components/SortBox.jsx

```diff
@@ -58,17 +58,15 @@ class SortBox extends Component {
           <button
             className={orderButtonClass}
             onClick={this.handleSortOrderChange}
+            title={formatMessage(
+              {
+                id: 'common.sort-box.sort'
+              },
+              { sortOrder }
```

Comment:
`sortOrder` got always english word `ascending` or `descending` taken from constant so it is not translated.